docs/source/Apps.md

```diff
@@ -1,18 +1,18 @@
-# More About Compliance Apps
+# Understanding Compliance Apps
 
-As described in the primary [README](README.md), Compliance apps map IT System components to compliance controls. A "component" can be any part of a system that contributes to its operation including organizational processes.
+Compliance apps map IT System components to compliance controls. A "component" can be any part of a system that contributes to its operation including organizational processes.
 
 Compliance apps collect and assess information about one or more system components and translate that information to compliance documentation.
 
 ![Apps map components to security/compliance controls](assets/app_diagram.png)
 
-Compliance apps can collect information about a system component from people (via web-based questionnaires) and from system components (via an [Automation API](Automation.md)).
+Compliance apps can collect information about a system component from people (via web-based questionnaires) and from system components (via an [Automation API](Automation.html)).
```

Comment:
Re-pointing the link targets to files that don't exist except for in the output of a build will make authoring more error prone because one can't tell if the link is correct or not until after building it.

Reply:
I agree, but in this first past, we don't have much of a choice. Recommonmark, the library that was suppose to fix has other problems. I'm willing to carry this as technical debt.